operations/configuration_management/CFS_Commands_Cheat_Sheet.md

```diff
@@ -52,6 +52,50 @@ For more information about the CFS API, see [Configuration Framework Service](..
     cray cfs v3 components describe <xname> --state-details true
     ```
 
+## Managing many components
+
+* (`ncn-mw#`) Enable or disable automatic configuration for a list of components:
+
+    ```bash
+    cray cfs v3 components updatemany --filter-ids <xname1>,<xname2>,... --enabled <true/false>
+    ```
+
+* (`ncn-mw#`) Set the desired configuration for a list of components:
+
+    ```bash
+    cray cfs v3 components updatemany --filter-ids <xname1>,<xname2>,... --desired-config <example-config>
+    ```
+
+* (`ncn-mw#`) Clear the state and trigger configuration on a list of components:
```

Comment:
I realize you were just copying the language used for the single-xname command above. I recommend making these edits to that command as well, because it applies there too.

```suggestion
* (`ncn-mw#`) Clear the state on a list of components:

    > For any of these components with a desired configuration set, this will trigger automatic configuration.
    > See [Automatic Configuration Management](Automatic_Configuration_Management.md) for more details.
```